app/Config/bootstrap.php

```diff
@@ -94,8 +94,8 @@
  * ));
  */
 Configure::write('Dispatcher.filters', array(
-	'AssetDispatcher',
-	'CacheDispatcher'
+    'AssetDispatcher',
```

Comment:
__[staging][code_sniffer]__
* Array value not aligned correctly; expected 40 spaces but found 4